src/analytics/types.ts

```diff
@@ -16,4 +16,5 @@ export type AnalyticsEventContext =
   | 'Ticket assistant'
   | 'Ticketing'
   | 'Trip details'
-  | 'Trip search';
+  | 'Trip search'
+  | 'InAppReview';
```

Comment:
I guess this should be `'In app review'` to follow the naming convention used elsewhere 🤓